app/ante_handler.go

```diff
@@ -6,8 +6,13 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
 	channelkeeper "github.com/cosmos/ibc-go/v2/modules/core/04-channel/keeper"
 	ibcante "github.com/cosmos/ibc-go/v2/modules/core/ante"
+
+	gaiaante "github.com/cosmos/gaia/v6/ante"
 )
 
+// TODO: Move this to the root ante package after this is backported to 6.x.x
```

Comment:
does this need to be backported? it would be state breaking so would only go into v8-Rho anyway, right?